libvmi/os/osx/memory.c

```diff
@@ -0,0 +1,147 @@
+/* The LibVMI Library is an introspection library that simplifies access to
+ * memory in a target virtual machine or in a file containing a dump of
+ * a system's physical memory.  LibVMI is based on the XenAccess Library.
+ *
+ * Copyright 2011 Sandia Corporation. Under the terms of Contract
```

Comment:
You don't need to add this Sandia Copyright to new files unless you copied portions of it from other files within the project.